myocamlbuild.ml

```diff
@@ -43,7 +43,22 @@ dispatch begin
       ~prods:["unix/%_stubs.c"; "unix/%_generated.ml"]
       ~deps: ["lib_gen/%_bindgen.byte"]
       (fun env build ->
-        Cmd (A(env "lib_gen/%_bindgen.byte")));
+         Cmd (S[A(env "lib_gen/%_bindgen.byte");
+                A"--c-file";
+                A(env "unix/%_stubs.c");
+                A"--ml-file";
+                A(env "unix/%_generated.ml")]));
+ 
+    rule "cstubs: lwt/x_bindings.ml -> lwt/x_stubs.c, lwt/x_generated.ml"
+      ~prods:["lwt/%_lwt_stubs.c"; "lwt/%_lwt_generated.ml"]
+      ~deps: ["lib_gen/%_bindgen.byte"]
```

Comment:
This is still not quite right (pattern doesn't match `prods` and `deps` but it doesn't *really* matter and I'd like to get the change in.